src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,15 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {comments.map(comment => (
+        <CommentInfo
+          comment={comment}
+          key={comment.id}
+        />
+      ))}
+    </div>
+  </>
```

Comment:
Don't add <> just in case
It is required only if you have more than 1 root element